features/providers/complete_and_check_means_answers.feature

```diff
@@ -48,5 +48,41 @@ Feature: Completing and checking means answers backwards and forwards
     Then I click 'Save and continue'
     Then I should be on a page showing "Check your answers"
 
+  @javascript @vcr
+  Scenario: I view the client completed means answers and I change vehicle questions
+    Given I start the journey as far as the client completed means page
+    Then I should be on a page showing 'Your client has completed their financial assessment'
+    Then I click 'Continue'
+    Then I should be on a page showing "Your client's income"
+    Then I click 'Save and continue'
+    Then I should be on a page showing "Your client's regular payments"
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'Check your answers'
```

Comment:
I wonder whether, if there was an "I start the journey as far as check your completed means page" step defined, this scenario could be broken down into smaller scenarios that each described one of the sub-journeys (Changing declared income, Change "Vehicles", etc.). That way we don't have such a large scenario where each sub-journey is so dependent on the success of the previous one.